extensions/amazon-lambda-http/http-event-server/src/main/java/io/quarkus/amazon/lambda/runtime/MockHttpEventServer.java

```diff
@@ -23,12 +23,11 @@
 
 public class MockHttpEventServer extends MockEventServer {
 
-    private final ObjectMapper objectMapper;
```

Comment:
Why removing this as a field?  Are you sure ObjectMapper won't get garbage collected?

Reply:
The field is never used, it never escapes the method that creates it